src/tools/yarprobotstatepublisher/src/robotstatepublisher.cpp

```diff
@@ -142,7 +142,7 @@ bool YARPRobotStatePublisherModule::configure(ResourceFinder &rf)
 
     // Setup the topic and configureisValid the onRead callback
     string jointStatesTopicName = rf.check("jointstates-topic",Value("/joint_states")).asString();
-    m_jointStateSubscriber = new JointStateSuscriber();
+    m_jointStateSubscriber = new JointStateSubcriber();
```

Comment:
`new` is a bad thing to use. As always, its better to replace it with smart pointers, if possible.

Reply:
https://github.com/robotology/idyntree/pull/535/commits/a4611e241fdf6704e0bdf125ee5e079858a40383